tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php

```diff
@@ -586,7 +586,7 @@ function validate(): void {}
  */
 function validate(): void {}
 ',
-            ['import_symbols' => true],
+            ['import_symbols' => true, 'phpdoc_tags' => ['link']],
```

Comment:
i would be wondering why `link` is not there by default, but probably other discussion :D